{"name": "test_new_customer_a1_error_2", "status": "passed", "description": "新增账户开户失败的用例", "attachments": [{"name": "log", "source": "34860060-b626-4801-8a16-c20f09cf48c8-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "f77abe5e-4eae-4f8d-a9c7-458117bd3224-attachment.txt", "type": "text/plain"}], "start": 1617349322898, "stop": 1617349324297, "uuid": "46c46e76-8364-4924-90dd-7cdbc5f77e94", "historyId": "439ad52a6dbf8a551db47b913e7abcc4", "testCaseId": "2abd7cf02e5ca8c24ecd7bae24728496", "fullName": "tests.test_a6_newclientregister.TestNewCustomerAccount#test_new_customer_a1_error_2", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_a6_newclientregister"}, {"name": "subSuite", "value": "TestNewCustomerAccount"}, {"name": "host", "value": "DESKTOP-U13POE8"}, {"name": "thread", "value": "6480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_a6_newclientregister"}]}